keras_core/ops/image.py

```diff
@@ -244,3 +244,216 @@ def affine_transform(
         fill_value=fill_value,
         data_format=data_format,
     )
+
+
+class ExtractPatches(Operation):
+    def __init__(
+        self,
+        size,
+        strides=None,
+        rates=1,
+        padding="valid",
+        data_format="channels_last",
+    ):
+        super().__init__()
+        self.size = size
+        self.strides = strides
+        self.rates = rates
+        self.padding = padding
+        self.data_format = data_format
+
+    def call(self, image):
+        return _extract_patches(
+            image,
+            self.size,
+            self.strides,
+            self.rates,
+            self.padding,
+            data_format=self.data_format,
+        )
+
+    def compute_output_spec(self, image):
```

Comment:
You should be able to reuse the same utilities that are used to compute the output spec of the `Conv` operation.